src/DateServiceProvider.php

```diff
@@ -18,9 +18,15 @@ class DateServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        $this->app['events']->listen('locale.changed', function () {
-            $this->setLocale();
-        });
+        if (version_compare(\App::version(), '5.5') >= 0) {
+            $this->app['events']->listen(LocaleUpdated::class, function() {
```

Comment:
You can't use `LocaleUpdated::class` notation since it should be imported and versions lower 5.5 will experience fail on this code. So, `\Illuminate\Foundation\Events\LocaleUpdated::class` should be used.

Reply:
Also, space should present between `function` and `()` to follow CS.